.github/workflows/build-and-test.yaml

```diff
@@ -103,6 +103,10 @@ jobs:
           elixir_version: "1.10.4"
           cmake_opts: "-DAVM_DISABLE_FP=on"
```

Comment:
Did you try to remove this option?
I believe this was a workaround for lack of extended allocator support.

Reply:
it would fail without it. While supporting FP instructions/operations in a simple way is not hard for AtomVM, -  it's not a goal of this PR.